skargo/src/main.sk

```diff
@@ -70,7 +70,11 @@ fun main(): void {
 }
 
 fun envCmd(args: Cli.ParseResults, fn: (Cli.ParseResults, Env) ~> void): void {
-  env = Env::create(args.getBool("verbose"));
+  env = Env::create(
+    // TODO: Properly support -q/--quiet for VerbosityQuiet, -v for VerbosityVerbose,
+    // and -vv for VerbosityExtraVerbose.
+    if (args.getBool("verbose")) VerbosityExtraVerbose() else VerbosityNormal(),
```

Comment:
Maybe factor this out into a function to reuse the eventual resolution of the TODO also for the cases below.